src/metabase/lib/js.cljs

```diff
@@ -418,6 +421,67 @@
   [a-query stage-number column]
   (lib.core/remove-field a-query stage-number column))
 
+(def ^:private visible-columns-defaults
+  {:include-joined?              true
+   :include-expressions?         true
+   :include-implicitly-joinable? true})
+
+
+;; TODO: Added as an expedient to fix metabase/metabase#32373. Due to the interaction with viz-settings, this issue
+;; was difficult to fix entirely within MLv2. Once viz-settings are ported, this function should not be needed, and the
+;; FE logic using it should be ported to MLv2 behind more meaningful names.
+(defn ^:export visible-columns
+  "Return a sequence of column metadatas for columns visible at the given stage of the query. Takes a map of options
+  which defines the set of columns to return. All the options default to *true*, ie. everything is returned by default.
+  ```
+  {
+    include_joined: true,
+    include_expressions true,
+    include_implicitly_joinable: true,
```

Comment:
Shouldn't these be in camelCase?

Reply:
btw @bshepherdson the FE don't use these options anymore, could we drop them entirely? so it would work like all of them are true